frontend/packages/dev-console/src/components/import/deployImage-submit-utils.ts

```diff
@@ -90,12 +95,18 @@ export const createOrUpdateImageStream = (
       ],
     },
   };
-
+  if (formData.resources === Resources.KnativeService && !_.isEmpty(originalImageStream)) {
+    const OrignalImageStreamTag = _.find(originalImageStream?.status?.tags, ['tag', tag]);
```

Comment:
```suggestion
    const orignalImageStreamTag = _.find(originalImageStream?.status?.tags, ['tag', tag]);
```
Not sure why this got a capital... it's just a value like anything else.

Reply:
Updated